### ISSUE TITLE
prepareAssembleJRubyStorm is not caching results

### ISSUE THREAD

**Issue by rtyler** [jruby-gradle/jruby-gradle-storm-plugin#22]
In my local testing, when I'm assembling the topology over and over it seems to be installing the gems over and over again.

It's not clear whether this is a storm plugin bug or a bug with the `JRubyPrepare` task

**Comment by rtyler** [comment]
It looks like something is modifying the installed gems causing the outputs to be marked as changed, triggering a reinstallation.

Guffaw!

```
:prepareAssembleJRubyStorm
Executing task ':prepareAssembleJRubyStorm' (up-to-date check took 0.049 secs) due to:
  Output file /home/tyler/source/lookout/REDACTED/build/dirinfo/jrubyStorm has changed.
  Output file /home/tyler/source/lookout/REDACTED/build/dirinfo/jrubyStorm/gems/configatron-2.13.0/spec/lib/class_spec.rb has been removed.
  Output file /home/tyler/source/lookout/REDACTED/build/dirinfo/jrubyStorm/gems/multi_json-1.11.2/README.md has been removed.

```

**Comment by rtyler** [comment]
I believe the info message above is only part of the story, if I'm interpreting [this line in the JRubyDirInfo constructor](https://github.com/jruby-gradle/jruby-gradle-plugin/blob/29bcf7a7337d2b5a0b643dcb196c151a94e53a3a/jruby-gradle-jar-plugin/src/main/groovy/com/github/jrubygradle/jar/internal/JRubyDirInfo.groovy#L18) then every time a new instance is being created (i.e. task construction) it will trash the entire dirinfo directory.

**Comment by rtyler** [comment]
The upstream issue has now been resolved with v1.0.4
